### PR TITLE
feat: added authentication pages

### PR DESCRIPTION
**Pull request by TejasShekar**
1. Added Login page
2. Added Sign Up page

### CODE REVIEW

File: Pages/authentication.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+.full-height {
+  height: 100%;
+}
+
+.brand-name h1{
+    font-size: 3rem;
+}
+
+.card{
+    max-width: 450px;
```
Comment (xii-vi) [inline Pages/authentication.css]: use relative unit

**Review by xii-vi — APPROVED** [review]
_Everything is in place. Go through comments before merging._

File: Pages/authentication.css (hunk 1 above)
Comment (TejasShekar) [reply Pages/authentication.css]: Done

**Review by BelideNikhil — APPROVED** [review]
navbar has logout button but it goes to login page
rest looks like it's Good to go!

**Pull request merged**